NEWS

```diff
@@ -11,6 +11,170 @@
+* Added proto v2 single zcopy (07152a5fb)
+* Added proto v2 query protocol description (49b0bbb91)
+* Added proto v2 AM short and bcopy (bbb954666)
+* Added UCP rcache (a8c5086bd)
+* Extended protocol configuration query (6e60f092b)
+* Improved adaptive progress - deactivate iface when discard/cleanup p2p lanes (582c6b41a)
+* Added error handling for protocols v2 (bbf640a53)
+* Added support of user memh in proto_v1 (58275b3d9)
+* Added support selecting local address, UCP_EP_PARAM_FIELD_LOCAL_SOCK_ADDR (bac14cafb)
+* Added option to limit GPUDirectRDMA size in rendezvous protocol, UCX_RNDV_MEMTYPE_DIRECT_SIZE (0442bbef6)
+* Added mem access lanes for all supported memory types (22117b304)
+* Disabled by default resolving remote EP ID when creating local EP (c538b7fc1)
+* Deprecated UCX_SOCKADDR_AUX_TLS configuration parameter (0ac6ce945)
+#### UCT
+* Introduced API uct_md_mkey_pack_v2 (8c744d744)
+* Introduced interfaces features API (ed22d5cd1)
+* Introduced max_inflight_eps parameter in perf_attr API (f90231db2)
+* Introduced API to resolve remote ID for EPs which support PEER_FAILURE and check peer existence (b56957fad)
+* Introduxed UCX_RCACHE_PURGE_ON_FORK (7f5677966)
+* Introduced iface.ops.ep_invalidate for testing purposes (33aaba968)
+#### RDMA CORE (IB, ROCE, etc.)
+* Introduced NDR autorecognition (bdd802c76, 9d75ac517)
+* Introduced CQE zipping support (34fd62b71, ab1353881)
+* Introduced ROCE_SUBNET_PREFIX_LEN handle for IB transports (72b962abf)
+* Set TM_MP_SRQ_ENABLE=try by default
+* Set the default MAX_RD_ATOMIC value to 16
+#### UCM
+* Introduced relative jumps translation in BISTRO (6c47694c0)
+#### UCS
+* Added topo module infrastructure (6912e1444)
+* Introduced memtrack output to VFS (b43e303d1)
+* Introduced rcache distribution to VFS (5081c9d2f)
+#### Tools
+* Added support pre-registered memory in ucx_perftest (8125b882c)
+* Added loopback transport support for UCT perf tests (446ac0a80)
+* Print memory invalidate cap by ucx_info -d (d0675935d)
+### Bugfixes
+#### Core
+* Fixed not deallocating memory from ucp_mem_unmap if no rcache (ff3fbc7d1)
+* Fixed compiler warnings with -Og option (7617e45ae)
+* Fixed versioning infrastructure (3cdd39f18, f5ca79c5c)
+* Multiple code improvements: refactoring, debug prints and assertions, etc (4198f9f51, 8bf6c1cfa, 287b1bb54, c3f310848, 2920e0c47, 24c74d10c, 03e9f2b00, 81ad07e90, bc1f1a0ac, 8db882ab1, 2a1def421, 672ac6e8b, cd016dd12, eb9a6b5c0, b2243a7b4, c4293983d, 76f1e4823, 865eca7db, 194ac47ac, 77cecb1e3, bb9101076, 050fae15a, 337dc62de, 2cc1efee0, b4e29e70e, 5083f56dc, 0ca0583ad, 830a91d46, ca3923b95, 28b3c26a1, 597d47124, e11d02b85, ab70ce3de, 622d6e038, 3bb649171, 76f1e4823, 572fea340, cc8260676, ea9922307, af3fc5a2c, 0ce726cd7, 76d9ed03e, bd81b941a, 7b4d154f4, 334072fe9, 2257b109a, 06af2d1fe, 5c564e1d4, 07c2f7b0d, c4293983d, 4b4a6976d, 0cfb0ab1b, e2a86a15a, 469705e87, 6ab76bf0b, 2a2c45387, e9a7c87d0, 7bafc7c20, d923167b1, 2cab571b1, ca57867dc, ec6f7af5c, 0d9060d2b)
```

Comment:
this list of commits is very long and does not help much..
IMO when should not list commits at all in NEWS